buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy

```diff
@@ -336,14 +336,26 @@ class GuideProjectGenerator implements AutoCloseable {
     }
 
     private static List<App> mergeApps(GuideMetadata base, GuideMetadata metadata) {
-        List<App> apps = new ArrayList<>(metadata.apps)
-        for (App app : apps) {
-            App baseAppMatchingName = base.apps.find { it.name == app.name }
-            if (baseAppMatchingName) {
-                app.features += baseAppMatchingName.features
-            }
+
+        Map<String, App> baseApps = base.apps.collectEntries { [(it.name): it] }
```

Comment:
Can you please add this change to a separate PR. So that we can merge it as soon as possible?

Reply:
https://github.com/micronaut-projects/micronaut-guides/pull/767